backend/core/detection.py

```diff
@@ -947,14 +988,16 @@ def commit_hijack(
                 serializer="yaml",
                 priority=0,
             )
-            hij_log.info(
-                "{}".format(
-                    json.dumps(hijack_log_field_formatter(result), cls=SetEncoder)
-                ),
-                extra={
-                    "community_annotation": result.get("community_annotation", "NA")
-                },
-            )
+            # log in hijack log only if it is a truly new ongoing hijack
+            if not outdated_parent:
```

Comment:
@slowr also about this? We can also log this as a new hijack and continue (its properties have changed anyway :) ).

Reply:
Actually let's log this but with the outdated_parent field enabled :). Resolving.